### PR TITLE
Add variables to group and layer models

### PR DESCRIPTION
**Pull request by Bo-Duke**
Ok so I'm not sure at ALL about some of the stuff here, especially around the migrations. I just applied `make migration` and `migrate`

### CODE REVIEW

File: project/terra_layer/models.py [hunk 1]
```diff
@@ -164,6 +167,8 @@ class LayerGroup(models.Model):
     )
     order = models.IntegerField(default=0)
     exclusive = models.BooleanField(default=False)
+    byVariable = models.BooleanField(default=False)
```
Comment (submarcos) [inline project/terra_layer/models.py:170]:
```suggestion
    by_variable = models.BooleanField(default=False)
```
respect the default python lower_case

File: project/terra_layer/views/layers.py [hunk 2]
```diff
@@ -466,6 +466,8 @@ def get_group_dict(self, group):
         group_content = {
             "group": group.label,
             "exclusive": group.exclusive,
+            "byVariable": group.byVariable,
```
Comment (submarcos) [inline project/terra_layer/views/layers.py:469]:
```suggestion
            "byVariable": group.by_variable,
```

File: project/terra_layer/models.py [hunk 3]
```diff
@@ -90,6 +90,8 @@ def tree2models(self, current_node=None, parent=None, order=0):
                 view=self,
                 label=current_node["label"],
                 exclusive=current_node.get("exclusive", False),
+                variables=current_node.get("variables", []),
+                byVariable=current_node.get("byVariable", False),
```
Comment (submarcos) [inline project/terra_layer/models.py:94]:
```suggestion
                by_variable=current_node.get("byVariable", False),
```
?

File: project/terra_layer/models.py (hunk 1 above)
Comment (Bo-Duke) [reply project/terra_layer/models.py]: ✅

File: project/terra_layer/models.py [hunk 4]
```diff
@@ -164,6 +167,8 @@ class LayerGroup(models.Model):
     )
     order = models.IntegerField(default=0)
     exclusive = models.BooleanField(default=False)
+    byVariable = models.BooleanField(default=False)
+    variables = models.JSONField(default=list)
```
[the remark this answers is not in the document]
Comment (Bo-Duke) [reply project/terra_layer/models.py]: ✅

**Pull request merged**